src/js/components/Nav/Nav.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Anchor } from '../Anchor';
+import { Box } from '../Box';
+
+// TODO resolve whether children should be allowed with items
+const Nav = ({ children, items = [], ...rest }) => (
+  <Box as="nav" flex={false} gap="medium" {...rest}>
+    {items.map(item => (
+      <Anchor href={item.href} label={item.label} key={item.label} />
```

Comment:
Why Anchor and not Button?

Reply:
I found Anchor to be more simplified than Button. 
You can leverage the children to show a Button instead (or even a Box), see for example, the CustomChild story on storybook, so the option for alternative components is still there.

Reply:
I made the component less opinionated by removing `items`.